src/body.js

```diff
@@ -433,34 +343,28 @@ export function extractContentType(body) {
  *
  * ref: https://fetch.spec.whatwg.org/#concept-body-total-bytes
  *
- * @param   Body    instance   Instance of Body
- * @return  Number?            Number of bytes, or null if not possible
+ * @param {any} obj.body Body object from the Body instance.
+ * @returns {number | null}
  */
-export function getTotalBytes(instance) {
-	const {body} = instance;
-
-	if (body === null) {
-		// Body is null
+export function getTotalBytes({body}) {
+	// Body is null or undefined
+	if (body == null) {
 		return 0;
 	}
 
+	// Body is Blob
 	if (isBlob(body)) {
 		return body.size;
 	}
 
+	// Body is Buffer
 	if (Buffer.isBuffer(body)) {
-		// Body is buffer
 		return body.length;
 	}
 
+	// Detect form data input from form-data module
 	if (body && typeof body.getLengthSync === 'function') {
-		// Detect form data input from form-data module
-		if (body._lengthRetrievers && body._lengthRetrievers.length == 0 || // 1.x
-			body.hasKnownLength && body.hasKnownLength()) { // 2.x
-			return body.getLengthSync();
-		}
-
-		return null;
+		return body.hasKnownLength && body.hasKnownLength() ? body.getLengthSync() : null;
```

Comment:
This introduced a breaking change: form-data v1 support is dropped here, please document in changelog and upgrade guide

Reply:
@bitinn It looks like the only thing that is a breaking change is that form-data v2 [drops support for NodeJS v0.10](https://github.com/form-data/form-data/releases/tag/v2.0.0). The first item in the changelog already documents the minimum version requirement.